src/core/ext/lb_policy/grpclb/grpclb.c

```diff
@@ -762,9 +795,9 @@ static int glb_pick(grpc_exec_ctx *exec_ctx, grpc_lb_policy *pol,
       GRPC_LB_POLICY_UNREF(exec_ctx, glb_policy->wc_arg.rr_policy, "glb_pick");
 
       /* add the load reporting initial metadata */
-      initial_metadata_add_lb_token(pick_args->initial_metadata,
-                                    pick_args->lb_token_mdelem_storage,
-                                    glb_policy->wc_arg.lb_token);
+      initial_metadata_add_lb_token(
+          pick_args->initial_metadata, pick_args->lb_token_mdelem_storage,
+          user_data_copy(glb_policy->wc_arg.lb_token));
```

Comment:
Now that you've changed the code to ensure that there's always a token (even if it's just `GRPC_MDELEM_LOAD_REPORTING_INITIAL_EMPTY`), I think that there's no more need for `user_data_copy()` to check whether the input is NULL.  As a result, it should be safe to replace that function with a direct call to `GRPC_MDELEM_REF()`.